src/it/resources-noncompilable/com/google/checkstyle/test/chapter4formatting/rule43onestatement/InputOneStatementPerLine.java

```diff
@@ -31,21 +34,35 @@ public class InputOneStatementPerLine {
   ;; // violation 'Only one statement per line allowed.'
   static {
     new JCheckBox().addActionListener((final ActionEvent e) -> { good(); });
+    // violation above ''{' at column 64 should have line break after.'
```

Comment:
All we need in file for validation is `Only one statement per line allowed.`
Why you keep unrelated?
Let's resolve them in any way.

Reply:
I have intentionally keep these violations. They're helping to cover edge cases and show that such type of statements are allowed.

Reply:
ok, lets keep it.